sor/src/main/java/com/bazaarvoice/emodb/sor/db/astyanax/AstyanaxDataReaderDAO.java

```diff
@@ -367,19 +371,45 @@ public Record read(Key key, ReadConsistency consistency) {
 
     @Timed (name = "bv.emodb.sor.AstyanaxDataReaderDAO.getSplits", absolute = true)
     @Override
-    public List<String> getSplits(Table tbl, int desiredRecordsPerSplit) {
+    public List<String> getSplits(Table tbl, int desiredRecordsPerSplit, int splitQuerySize) throws TimeoutException {
```

Comment:
There's no unit test specifically for this method, and given its complicated nature I'd be more comfortable if there were one.  I understand it's complicated to do since the underlying call to `getCfSplits()` requires an actual Cassandra connection to test properly.  Maybe refactor such that this method can be tested independently of the call to `getCfSplits` and then open up as `@VisibleForTesting` so that at least one unit test can verify it's resplitting correctly.